src/t2iapi/context/service.proto

```diff
@@ -158,6 +158,52 @@ service ContextService {
   rpc EnsembleContextIndicateMembershipWithIdentification (BasicHandleRequest)
       returns (EnsembleContextIndicateMembershipWithIdentificationResponse);
 
+  /*
+  Indicate membership in an SDC PARTICIPANT ENSEMBLE resp. in SDC PARTICIPANT ENSEMBLEs for a provided
+  pm:EnsembleContextDescriptor by associating each state contained in a representative set of pm:EnsembleContextStates
+  of that descriptor. Return the list of handles of states contained in that representative set.
+
+  The representative set of pm:EnsembleContextStates for a given pm:EnsembleContextDescriptor is a minimal set of
+  pm:EnsembleContextStates which if associated are sufficient to indicate membership in the SDC PARTICIPANT ENSEMBLE(s)
+  for that descriptor.
+
+  Example 1: treatment session
+    A POC MEDICAL DEVICE currently working in a (temporal) logical group based on a treatment session
+    may expressed by a pm:EnsembleContextDescriptor. If each of its pm:EnsembleContextStates describes a step in
```

Comment:
Better: "The fact that a POC MEDICAL DEVICE is currently working in a (temporal) logical group based on a treatment session may be expressed by a pm:EnsembleContextDescriptor"

Reply:
done